downstream/modules/platform/proc-aap-controller-backup.adoc

```diff
@@ -0,0 +1,42 @@
+[id="aap-controller-backup"]
+
+= Backing up the {ControllerNameStart} deployment
+
+[role=_abstract]
+Use this procedure to back up a deployment of the controller, including jobs, inventories, and credentials.
+
+.Prerequisites
+
+* You must be authenticated with an Openshift cluster.
+* The {OperatorPlatform} has been deployed to the cluster.
```

Comment:
nitpick, this should say installed, not deployed:

```
* The {OperatorPlatform} has been installed to the cluster.
```

(_install_ the operator, _deploy_ the app components like controller and hub)

Reply:
Fixed.